conformity/account_settings.go

```diff
@@ -442,13 +515,90 @@ func processRuleValues(vs []interface{}) []*cloudconformity.RuleSettingValues {
 	values := make([]*cloudconformity.RuleSettingValues, 0, len(vs))
 	for _, v := range vs {
 
-		profileValues := &cloudconformity.RuleSettingValues{}
+		ruleSettingValues := &cloudconformity.RuleSettingValues{}
 		item := v.(map[string]interface{})
 
-		profileValues.Enabled = item["enabled"].(bool)
-		profileValues.Label = item["label"].(string)
-		profileValues.Value = item["value"].(string)
-		values = append(values, profileValues)
+		ruleSettingValues.Enabled = item["enabled"].(bool)
+		ruleSettingValues.Label = item["label"].(string)
+		ruleSettingValues.Value = item["value"].(string)
+
+		values = append(values, ruleSettingValues)
+	}
+	return values
+}
+
+func processRuleValuesSettings(vs []interface{}) []*cloudconformity.RuleSettingValues {
+	values := make([]*cloudconformity.RuleSettingValues, 0, len(vs))
+	for _, v := range vs {
+
+		ruleSettingValues := &cloudconformity.RuleSettingValues{}
+		item := v.(map[string]interface{})
+
+		if val, ok := item["enabled"]; ok && val != nil {
+			ruleSettingValues.Enabled = val.(bool)
+		}
+
+		if val, ok := item["customised"]; ok && val != nil {
+			ruleSettingValues.Customised = val.(bool)
+		}
```

Comment:
❓ Would doing the type cast in the if statement be safer? or does that cause issues since `bool` can never be `nil`?
```suggestion
		if val, ok := item["enabled"].(bool); ok && val != nil {
			ruleSettingValues.Enabled = val
		}

		if val, ok := item["customised"].(bool); ok && val != nil {
			ruleSettingValues.Customised = val
		}
```

Reply:
@johan1252 
These 2 fields are optional and can be ignored in the template so I use this way.

Reply:
Okay understood, thanks